src/components/modals/ContactsPermissionModal.js

```diff
@@ -61,15 +60,12 @@ const getStyles = cacheStyles((theme: Theme) => ({
     alignItems: 'center'
   },
   header: {
-    marginVertical: theme.rem(1),
+    marginVertical: theme.rem(0.5),
     fontFamily: theme.fontFaceMedium,
     fontSize: theme.rem(1.25),
     includeFontPadding: false
   },
   message: {
     marginBottom: theme.rem(1)
-  },
```

Comment:
This is not the solution, we do want to keep the margin at the bottom if it fits in the screen.